components/bank/BankWorst.vue

```diff
@@ -42,20 +35,13 @@ import { PrismicDocument } from '@prismicio/types'
 const props = defineProps<{
     name: string,
     website: string,
-    subtitle: string,
-    header: string,
-    summary: string,
-    details: string
     inheritBrandRating: {
         name: string,
         tag: string
     },
-    amountFinancedSince2016: string,
     bankPage: PrismicDocument<Record<string, any>, string, string> | null
 }>()
 
-const formattedTotal = computed(() => props.amountFinancedSince2016 ?? 'large amounts')
-
 const piggyText =
-    `While you’ve been stashing away money for a house or a weekend get-away, ${props.name} has been using your savings to lend to some very questionable fossil fuel friends.\n\nAnd it's not just a little here and there, we’re talking about ${formattedTotal.value || "hundreds of millions to hundreds of billions of US dollars"} in the 7 years since 197 countries agreed to drastically reduce their greenhouse gas emissions in the Paris Agreement.`
```

Comment:
Text should stay as is here. Eventually we'll want to migrate the text to prismic but but that's a different ticket/pr